helix-core/src/main/java/org/apache/helix/tools/ClusterVerifiers/ZkHelixClusterVerifier.java

```diff
@@ -48,6 +50,10 @@
   private static Logger LOG = LoggerFactory.getLogger(ZkHelixClusterVerifier.class);
   protected static int DEFAULT_TIMEOUT = 300 * 1000;
   protected static int DEFAULT_PERIOD = 500;
+  // COOL_DOWN before starting vefiyByPool
+  // The goal is to make sure waiting for controller pipeline starts at least one cycle
+  // to update ideal state.
+  protected static int DEFAULT_COOLDOWN = 2 * 1000;
```

Comment:
Set a hardcoded time here is not a good idea, these tools have been used by many of customers, including Espresso, how do we know 2 second is good enough?

Reply:
This is just by experiment. Quite some flaky ones are gone. 

In fact, longer cool_down aside from making test running a little bit longer, no negative impact, only positive impact. From the github run, this does not really significant (not noticable) make the test running longer,

Reply:
Changed the way to what we discussed before.